### ISSUE TITLE
GuiValueBox() ERROR: RLGL: Batch elements overflow

### ISSUE THREAD

**Issue by BenStigsen** [raysan5/raygui#127]
```c
int value = 1000;
bool editable = true;

if (GuiValueBox((Rectangle){600, 50, 100, 50}, "Value", &value, 0, 10000, editable ))  {
    editable = !editable;
}
```
When it first contains the content it says "1000", then I press backspace, until I reach "1", then it crashes.

**Comment by raysan5** [comment]
@BenStigsen Just added an additional security check on raylib `DrawTexturePro()` to avoid this kind of overflow, it should work with latest raylib.

**Comment by BenStigsen** [comment]
Sorry for the late response.
This issue hasn't been fixed in the latest Raylib version (3.7.0 - 26. April).

**Comment by raysan5** [comment]
Just tested it on `controls_test_suite.c` and it didn't crash for me.
